scripts/newinstall.sh

```diff
@@ -636,15 +636,10 @@ n8l::miniconda::bootstrap() {
 	if [[ -z $miniconda_path ]]; then
 		local miniconda_base_path="${prefix}/conda"
 		miniconda_path="${miniconda_base_path}/$(n8l::miniconda_slug)"
-		echo "Installing conda at ${miniconda_path}"
-		do_install=true
-	else
-		# Check commands in supplied conda environment
-		(
-			export PATH="${miniconda_path}/bin:${PATH}"
-			n8l::require_cmds conda
-		)
-		echo "Using conda at ${miniconda_path}"
+		if [[ ! -e $miniconda_path ]]; then
+			echo "Installing conda at ${miniconda_path}"
+			do_install=true
+		fi
 	fi
 
 	if [[ $do_install == true ]]; then
```

Comment:
Since this can only be set by one path, perhaps it should be merged?

Reply:
How about something like this?

It's a bit weird, but I want to make sure the messages match what is actually happening (otherwise I need to redo unit tests a bit too).

```bash
	local performed_install=false

	# Clear arguments for source
	while (( "$#" )); do
		shift
	done

	if [[ -z $miniconda_path ]]; then
		local miniconda_base_path="${prefix}/conda"
		miniconda_path="${miniconda_base_path}/$(n8l::miniconda_slug)"
		if [[ ! -e $miniconda_path ]]; then
			echo "Installing conda at ${miniconda_path}"
			n8l::miniconda::install \
				"$mini_ver" \
				"$miniconda_path" \
				"$miniconda_base_url"
			# only miniconda current symlink if we installed miniconda
			n8l::ln_rel "$miniconda_path" current
			performed_install = true
		fi
	fi

	if [[ $performed_install == false ]]; then
		(
			export PATH="${miniconda_path}/bin:${PATH}"
			n8l::require_cmds conda
		)
		echo "Using conda at ${miniconda_path}"
	fi
```